Multisig/UI/Safe Management/Add Safe/SafeLoadedViewController/SafeLoadedViewController.swift

```diff
@@ -43,8 +43,15 @@ class SafeLoadedViewController: UIViewController {
 
     @IBAction func importOwnerButtonTouched(_ sender: Any) {
         Tracker.shared.track(event: TrackingEvent.userOnboardingOwnerImport)
-        let vc = ViewControllerFactory.importOwnerViewController(presenter: self)
-        present(vc, animated: true)
+
+        var controller: UIViewController
+        if App.configuration.toggles.walletConnectOwnerKeyEnabled {
+            controller = ViewControllerFactory.selectKeyTypeViewController(presenter: self)
+        } else {
+            controller = ViewControllerFactory.importOwnerViewController(presenter: self)
+        }
```

Comment:
refactor with the newly added "how to add key" screen

Reply:
done